airflow/utils/operator_helpers.py

```diff
@@ -75,6 +86,16 @@ def context_to_airflow_vars(context: Mapping[str, Any], in_env_var_format: bool
         (dag_run, 'run_id', 'AIRFLOW_CONTEXT_DAG_RUN_ID'),
     ]
 
+    context_params = settings.get_airflow_context_vars(context)
+    for key, value in context_params.items():
+        if in_env_var_format:
+            if not key.startswith(ENV_VAR_FORMAT_PREFIX):
+                key = ENV_VAR_FORMAT_PREFIX + key.upper()
+        else:
+            if not key.startswith(DEFAULT_FORMAT_PREFIX):
+                key = DEFAULT_FORMAT_PREFIX + key
+        params[key] = value
```

Comment:
Some validation is needed to ensure the dict still only contain str.

Reply:
Agree. Sanity check here is a good thing.

Reply:
good point. @potiuk @uranusjr addressed the comment.

Reply:
Should probably mention this in the documentation page as well.